pkg/kudoctl/cmd/root.go

```diff
@@ -25,6 +25,9 @@ and serves as an API aggregation layer.
 	# View all plan history of a specific package
 	kubectl kudo plan history [flags]
 
+	# Run a test suite
```

Comment:
maybe be more specific? Test suite of what? Tests run against your cluster and running kudo? Testing kudo? Testing frameworks?

Reply:
Changed. You can run any test suite that you provide to it.